public/scripts/suggestion.js

```diff
@@ -0,0 +1,17 @@
+$( document ).ready(function() {
+ $('#git_url').editableSelect();
+ $('#git_url').keyup(() => {
+   let q = $("#git_url").val()
```

Comment:
I habituated to write code with let and without a semicolon. I'll change everywhere. :sweat_smile: 
`q` for query and it is widely used everywhere. even I'll change to query